flank-scripts/src/main/kotlin/flank/scripts/contribution/ideaktlint/IdeaKtlintCodeStyleCommand.kt

```diff
@@ -0,0 +1,48 @@
+package flank.scripts.contribution.ideaktlint
+
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.kittinunf.fuel.httpDownload
+import flank.common.deleteFile
+import flank.common.logLn
+import flank.scripts.utils.SUCCESS
+import flank.scripts.utils.runCommand
+import kotlinx.coroutines.runBlocking
+import java.io.File
+
+object IdeaKtlintCodeStyleCommand : CliktCommand(
+    name = "applyKtlintToIdea",
+    help = "Applies Ktlint to this idea project forcefully"
+) {
+    override fun run(): Unit = runBlocking {
+        logLn("Applying Ktlint code style to this idea project")
+        logLn("Retrieving Ktlint...")
+        when (retrieveKtlintCurl()) {
```

Comment:
Why 2 ways of downloading ktlint? Can't we just use `retrieveKtlintResolve` ?

Reply:
Windows doesnt always have a bash or mingw backend therefore curl isnt always avaialble. The second is a backup 👍

Reply:
Got it. How about using `retrieveKtlintResolve` only? A bit less logic and it's working regardless of OS

Reply:
@pawelpasterz i will test it :) I was looking for a nice progress bar. Doing it with Fuel creates way more logic and for me specifically i have curl installed on windows so it generally looks good. Its for edge cases.

Reply:
have bigger issues with pathing at the moment than anything else.

Reply:
@Sloox  if it is possible please use the non-curl version for all OS.

Reply:
+1 to investing in the Kotlin logic that works for all OS instead of depending on curl.

Reply:
Ill remove the curl command